### PR TITLE
#732 Update Travis URL on README

### LINKED ISSUE #732: [bincrafters-conventions] Update Travis URL
### Description of your feature

Since Bincrafters are using travis.com, many recipes contain travis.org on README file. The application Bincrafters Conventions could update the URL to use .com

### PR DESCRIPTION
**Pull request by uilianries**
closes https://github.com/bincrafters/community/issues/732

### CODE REVIEW

File: bincrafters_conventions/bincrafters_conventions.py [hunk 1]
```diff
@@ -134,6 +137,7 @@ def run(self, *args):
             if os.path.isfile("appveyor.yml"):
                 self._update_appveyor_file("appveyor.yml")
             self._update_conanfile("conanfile.py")
+            self._update_readme("README.md")
```
Comment (Croydon) [inline bincrafters_conventions/bincrafters_conventions.py]:
```suggestion
            if os.path.isfile("README.md"):
                self._update_readme("README.md")
```

File: bincrafters_conventions/actions/update_travis_url.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+def update_travis_url(main, file):
```
Comment (Croydon) [inline bincrafters_conventions/actions/update_travis_url.py]: Could we please rename this function + file to `update_readme_travis_url`? I would like to have them prefix with the category of file(s) it is manipulating

File: bincrafters_conventions/actions/update_travis_url.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    """ Update Travis CI URL
+    """
+    if main.replace_in_file(file, "https://travis-ci.org/bincrafters", "https://travis-ci.com/bincrafters"):
+        main.output_result_update(title="README: Update Travis URL")
```
Comment (Croydon) [inline bincrafters_conventions/actions/update_travis_url.py]:
```suggestion
        main.output_result_update(title="README: Update Travis URL from .org to .com")
```

**Review by Croydon — CHANGES_REQUESTED** [review]

File: bincrafters_conventions/actions/update_travis_url.py (hunk 2 above)
Comment (uilianries) [reply bincrafters_conventions/actions/update_travis_url.py]: yes, it sounds better. I'll update it.

Comment (uilianries) [reply bincrafters_conventions/actions/update_travis_url.py]: done!

**Review by Croydon — APPROVED** [review]
LGTM

**Pull request merged**